wordpress-mobile/WordPress-Android
# Sort more than 100 pages chronologically

Fixes #10440 

This PR copies the Calypso behaviour where it sorts more than 100 published pages chronologically instead of topologically. In this view we're not showing parent/child relationship between the pages. I've also added unit tests to cover this use case.  

To easier test this you can the `MAX_TOPOLOGICAL_PAGE_COUNT` to a more reasonable number like 10. 

To test:
* Go to a site with < 100 published pages
* Check that they are still ordered topologically 
* Go to a site with >= 100 published pages
* Check that they are ordered chronologically in a descending order by the published date. 
* Check that you can't see the parent/child relationship

Update release notes:
